code/__DEFINES/misc.dm

```diff
@@ -13,36 +13,38 @@
 
 //Human Overlays Indexes/////////
-#define MUTATIONS_LAYER 29 //mutations. Tk headglows, cold resistance glow, etc
-#define BODY_BEHIND_LAYER 28 //certain mutantrace features (tail when looking south) that must appear behind the body parts
-#define BODYPARTS_LAYER 27 //Initially "AUGMENTS", this was repurposed to be a catch-all bodyparts flag
-#define BODY_ADJ_LAYER 26 //certain mutantrace features (snout, body markings) that must appear above the body parts
-#define BODY_LAYER 25 //underwear, undershirts, socks, eyes, lips(makeup)
-#define FRONT_MUTATIONS_LAYER 24 //mutations that should appear above body, body_adj and bodyparts layer (e.g. laser eyes)
-#define DAMAGE_LAYER 23 //damage indicators (cuts and burns)
-#define UNIFORM_LAYER 22
-#define ID_LAYER 21
-#define ID_CARD_LAYER 20
-#define HANDS_PART_LAYER 19
-#define GLOVES_LAYER 18
-#define SHOES_LAYER 17
-#define EARS_LAYER 16
-#define SUIT_LAYER 15
-#define GLASSES_LAYER 14
-#define BELT_LAYER 13 //Possible make this an overlay of somethign required to wear a belt?
-#define SUIT_STORE_LAYER 12
-#define NECK_LAYER 11
-#define BACK_LAYER 10
-#define HAIR_LAYER 9 //TODO: make part of head layer?
-#define FACEMASK_LAYER 8
-#define HEAD_LAYER 7
-#define HANDCUFF_LAYER 6
-#define LEGCUFF_LAYER 5
-#define HANDS_LAYER 4
-#define BODY_FRONT_LAYER 3
+#define MUTATIONS_LAYER 31 //mutations. Tk headglows, cold resistance glow, etc
+#define BODY_BEHIND_LAYER 30 //certain mutantrace features (tail when looking south) that must appear behind the body parts
+#define BODYPARTS_LAYER 29 //Initially "AUGMENTS", this was repurposed to be a catch-all bodyparts flag
+#define BODY_ADJ_LAYER 28 //certain mutantrace features (snout, body markings) that must appear above the body parts
+#define BODY_LAYER 27 //underwear, undershirts, socks, eyes, lips(makeup)
+#define FRONT_MUTATIONS_LAYER 26 //mutations that should appear above body, body_adj and bodyparts layer (e.g. laser eyes)
+#define DAMAGE_LAYER 25 //damage indicators (cuts and burns)
+#define UNIFORM_LAYER 24
+#define ID_LAYER 23
+#define ID_CARD_LAYER 22
+#define HANDS_PART_LAYER 21
+#define GLOVES_LAYER 20
+#define SHOES_LAYER 19
+#define EARS_LAYER 18
+#define SUIT_LAYER 17
+#define GLASSES_LAYER 16
+#define BELT_LAYER 15 //Possible make this an overlay of somethign required to wear a belt?
+#define SUIT_STORE_LAYER 14
+#define NECK_LAYER 13
+#define BACK_LAYER 12
+#define HAIR_LAYER 11 //TODO: make part of head layer?
+#define FACEMASK_LAYER 10
+#define HEAD_LAYER 9
+#define HANDCUFF_LAYER 8
+#define LEGCUFF_LAYER 7
+#define HANDS_LAYER 6
+#define BODY_FRONT_LAYER 5 // Usually used for mutant bodyparts that need to be in front of stuff (e.g. cat ears)
+#define ABOVE_BODY_FRONT_GLASSES_LAYER 4 // For the special glasses that actually require to be above the hair (e.g. lifted welding goggles)
+#define ABOVE_BODY_FRONT_HEAD_LAYER 3 // For the rare cases where something on the head needs to be above everything else (e.g. flowers)
```

Comment:
god we really need to change these to like, reference each other

so like `#define ABOVE_BODY_FRONT_HEAD_LAYER (ABOVE_BODY_FRONT_GLASSES_LAYER - 1)`

that way we dont have these horrific diffs

Reply:
That's a good idea, I was trying to think of a smarter way to do this as well. That'd be a lot better. Should I do that in this PR, or in the next one? It's not a big refactor, but I wouldn't want to cause conflicts with other stuff.